xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -3881,6 +3881,7 @@
         "firstaired",
         "season",
         "episode",
+        "episodepart",
```

Comment:
This change either has to go into your other PR (which I suggest) or json-rpc API needs to be bumped in this PR.

Reply:
OK I consider that part of the "missing" EpisodePart information but that is fine they are pretty tightly coupled.